### DISCUSSION TITLE
如果要适配720p，需要改什么？

### DISCUSSION THREAD

**Discussion by CallMeChin** [CHNZYX/Auto_Simulated_Universe#174] [Q&A]
适配720p有什么困难吗？如果能该改什么呢？

**Comment by CHNZYX** [comment]
图片匹配会出问题，不是缩放图片那么简单，因此需要重新截图。游戏内组件排版会有变化，因此代码方面也需要适配。因为ocr不受分辨率影响，因此适配工作量会降低一些，但目前没有很大的需求适配就没做了
